projects/biometrics/index.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: page
+title: Biometric Privacy
+image: images/biometrics/titleimage.jpg
+permalink: /projects/biometrics/
+---
+
+![protest](/images/biometrics/protest.jpg)
+---
+---
+---
+---
```

Comment:
Maybe use either zero or one horizontal lines (`---`).

Reply:
The layout looks terrible without the four (technically only three render)